package.json

```diff
@@ -50,8 +50,7 @@
     "freedom": "^0.6.11"
   },
   "scripts": {
-    "test": "grunt test",
-    "prepublish": "grunt test",
+    "test": "grunt build jasmine",
```

Comment:
Neither: Chrome isn't available in the Travis environment so the integration tests can't be run. That's the only reason.